privacyidea/static/components/token/views/token.import.html

```diff
@@ -49,7 +49,7 @@ <h3 translate>Import tokens from a file</h3>
             The file is supposed to contain one token per line.
         </p>
         <p class="help-block" translate>
-            If you enrolling the Yubikey in HOTP mode, you should use the
+            If you are enrolling the Yubikey in HOTP mode, You should use the
```

Comment:
"You should use" shouldn't it be a lower case "y"?

Reply:
@plettich I took the liberty to change this.

Reply:
yeah, i guess i am old fashioned, i always capitalize the form of address :-)